router.php

```diff
@@ -1,18 +1,21 @@
 <?php
 // Used by `wp server` to route requests.
 
-namespace WP_CLI\Router;
+namespace WP_CLI\Server\Router;
```

Comment:
It is safer to keep the nameserver unchanged.
```suggestion
namespace WP_CLI\Router;
```